src/BaseFactory.php

```diff
@@ -74,13 +65,21 @@ public function times(int $times = 1): MultiFactoryCollection
         }));
     }
 
+    /**
```

Comment:
Please remove the docBlock. I saw some other methods have them now too, but probably due some other PRs. I prefer `not` to use them when using types for parameters and the return.

Reply:
The Phpdoc is needed to help the IDE autocomplete. The `static` return type keyword will be introduced in PHP 8 and will fill the gap with self.

Reply:
Do you prefer if I only use `/** @return static */` ?

Reply:
Which IDE do you use that you need that "@return $this"? In PhpStorm I don't need when using the `self` return type. But I know there was a problem with this in VS Code so we added the `static` return in the docblock just for VS Vode. (https://github.com/christophrumpel/laravel-factories-reloaded/pull/24)

Reply:
When I tested, I was indeed using VSCode.

Reply:
Ok, so if it helps with auto-completing in VS code, we can keep it.  For PhpStorm I know that the hinted return type self does work currently as long as there is not "static" return type yet.